esprima.js

```diff
@@ -4095,6 +4100,56 @@
         return list;
     }
 
+
+    function tokenizerState() {
```

Comment:
It's necessary. See the two token lookahead in [section 13.5](http://www.ecma-international.org/ecma-262/6.0/index.html#sec-expression-statement) and the associated NOTE text.